test/docs.js

```diff
@@ -111,10 +111,17 @@ describe('documentation', function() {
       global: global
     };
 
+    function FakeExpress() {
+      return {
+        get: function() {}
```

Comment:
If mitm did allow such a thing (it probably does), a response that works for all requests wouldn't be possible. And there's still the error handling that would get skipped. We would get to a point of managing a response for each method, that we've crossed over into the territory of what our unit tests do.

If you think going down that road is worth it, go ahead and open up an issue!